src/coreclr/jit/importer.cpp

```diff
@@ -3860,6 +3860,31 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
                 retNode = impArrayAccessIntrinsic(clsHnd, sig, memberRef, readonlyCall, ni);
                 break;
 
+            case NI_System_String_Equals:
```

Comment:
If we see `NI_System_MemoryExtensions_AsSpan` or `NI_System_String_op_Implicit` cases here (assuming it is possible), can we reuse something from `impGetStrConFromSpan()`?

Reply:
@am11 not sure I follow - what do you mean?

Reply:
ah you mean can we do something for them here? I don't think so, string literal to Span is already pretty good:
![image](https://user-images.githubusercontent.com/523221/154337766-1df3977c-8beb-48a0-b3de-82f30c9d9040.png)

Reply:
Do we need to handle `case NI_System_MemoryExtensions_AsSpan` and `case NI_System_String_op_Implicit` here as well?